util/session/sessionmanager_test.go

```diff
@@ -221,6 +225,146 @@ func TestSessionManager_ProjectToken(t *testing.T) {
 	})
 }
 
+type claimsMock struct {
+	err error
+}
+
+func (cm *claimsMock) Valid() error {
+	return cm.err
+}
```

Comment:
I think this method and the `err` field can be dropped now that we're not doing claims validation at the claims struct level.

Reply:
the method needs to be there so it correctly implements the interface